app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
+end
```

Comment:
- [ ] Lastly again for the Like model kindly add the Rails Active Call Back as explained for the Comment model

Reply:
Thank you very much, I will make changes as suggested